src/Common/ruleset.xml

```diff
@@ -18,6 +18,13 @@
         <priority>1</priority>
     </rule>
 
+    <rule
+            name="ImplementsApiInheritDocRule"
+            message="Access: {0}"
+            class="ArchitectureSniffer\Common\ImplementsApiInheritDocRule">
+
+        <priority>3</priority>
```

Comment:
@dereuromark Could you please help selecting the priority?

Reply:
3 is fine, we need to then check if we can ignore/whitelist certain modules.